src/foam/nanos/hello/tests.jrl

```diff
@@ -0,0 +1,2 @@
+p({"class":"foam.nanos.hello.HelloTest","id":"HelloTest"})
+p({"class":"foam.nanos.test.Test","id":"HelloTestJS","server":false,"status":2,"code":"var hello = foam.nanos.hello.ClientHelloService.create({\ndelegate: foam.box.HTTPBox.create({\nurl: 'service/hello'\n}, x)\n}, x);\nhello.hello(\"JS\").then(function(msg) { foam.assert(msg === (\"helloJS\"), \"ClientHelloService succeed\") });"})
```

Comment:
Sorry I didn't provide enough detail.  In nanos tests we want to use test(condition, message) rather than foam.assert.

So test(msg == 'whatever', "Didn't received expected greeting.") for example.